new events - ready for stats v2.5

killer2k1 opened the pull request.

killer2k1 committed:
more events

clutch return, egrab

killer2k1 committed:
Merge branch 'tribesonecommunity:master' into master

killer2k1 committed:
reuploading

next server update

dwayneyuen commented on base/scripts/code.game.cs:
```diff
@@ -1139,4 +1139,31 @@ function remoteSetArmor(%player, %armorType)
+    }
+    //otherwise position of stand
+    else {
+        %flagPos = $teamFlagStandPos[%flagTeam];
```
there exists a global we can already use here:

```
$teamFlag[%flagTeam].originalPosition;
```

dwayneyuen commented on base/scripts/objectives.cs:
```diff
@@ -1071,22 +1090,47 @@ function checkObjectives(%this)
+        //if flag is at home - on stand
+        //check radius distance to their own flag
+        
+        %doNotCheck = false;
```
i think we can use `%flag.atHome` here

edit: looking back, i guess since you have to enumerate those two cases anyway, it doesn't save us anything, so up to you

killer2k1 committed:
tweaked radius fixed logic error

egrab = 30
clutch return = 15

else statement on clutch return now checks for enemy flag distance to its own stand
